libs/features/mas/commerce/libs/commerce.d.ts

```diff
@@ -608,9 +595,9 @@ export declare const getSettings: Commerce.getSettings;
  * init(getConfig);
  * init(getConfig, true);
  */
-export declare const init: Commerce.initService;
+export declare const init: Commerce.init;
 /**
  * Disposes active instance of `Commerce` service allowing it to be initialised again
  * via normal call to `init`.
  */
-export declare const reset: Commerce.resetService;
+export declare const reset: Commerce.reset;
```

Comment:
I think we should not support resetting the service. It increasing the complexity.
with the custom element, to reset, one should remove it from DOM and add a new one.